interface/modules/zend_modules/module/Installer/view/installer/installer/index.phtml

```diff
@@ -32,7 +32,7 @@ $zendModDir 	= $GLOBALS['zendModDir'];
 $depObj 	= $this->dependencyObject;
 
 ?>
-<h1><img src="<?php echo $this->basePath()?>/images/module.png" class="nav-menu-img"><?php echo $this->escapeHtml($title); ?></h1>
+<h1><img src="<?php echo $GLOBALS['images_static_relative']; ?>/module.png" class="nav-menu-img"><?php echo $this->escapeHtml($title); ?></h1>
```

Comment:
will need to revert above line. The zend_modules directory has its own assets/images (`$this->basePath()` is not going to the main images directory)